dotcom-rendering/README.md

```diff
@@ -2,7 +2,21 @@
 
 Frontend rendering framework for theguardian.com. It uses [React](https://reactjs.org/), with [Emotion](https://emotion.sh) for styling.
 
-## Quick start
+## Quick start for first time dev install
+
+```sh
+$ corepack enable
+$ asdf reshim nodejs # if you're using asdf, this step is needed afterwards
+$ git clone git@github.com:guardian/dotcom-rendering.git
+$ nvm use # not needed if you're using asdf
+$ pnpm install --frozen-lockfile
+$ cd dotcom-rendering
+$ make dev
+```
+
+Then open [http://localhost:3030](http://localhost:3030)
```

Comment:
This will be shown in the console. Until then, or if anything fails above, of if the port changes in the future, this link will not work.